src/SFA.DAS.LevyTransferMatching/Application/Queries/Pledges/GetApplicationsForDownload/GetApplicationsForDownloadQueryHandler.cs

```diff
@@ -0,0 +1,154 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using MediatR;
+using SFA.DAS.LevyTransferMatching.InnerApi.Responses;
+using SFA.DAS.LevyTransferMatching.Interfaces;
+using SFA.DAS.LevyTransferMatching.Models.ReferenceData;
+using SFA.DAS.SharedOuterApi.Configuration;
+using SFA.DAS.SharedOuterApi.InnerApi.Requests;
+using SFA.DAS.SharedOuterApi.InnerApi.Responses;
+using SFA.DAS.SharedOuterApi.Interfaces;
+using SFA.DAS.SharedOuterApi.Models;
+
+namespace SFA.DAS.LevyTransferMatching.Application.Queries.Pledges.GetApplicationsForDownload
+{
+    public class GetApplicationsForDownloadQueryHandler : IRequestHandler<GetApplicationsForDownloadQuery, GetApplicationsForDownloadQueryResult>
+    {
+        private readonly ICoursesApiClient<CoursesApiConfiguration> _coursesApiClient;
+        private readonly ILevyTransferMatchingService _levyTransferMatchingService;
+        private readonly IReferenceDataService _referenceDataService;
+        private List<ReferenceDataItem> _allJobRoles;
+        private List<ReferenceDataItem> _allLevels;
+        private List<ReferenceDataItem> _allSectors;
+
+
+        public GetApplicationsForDownloadQueryHandler(ICoursesApiClient<CoursesApiConfiguration> coursesApiClient, ILevyTransferMatchingService levyTransferMatchingService, IReferenceDataService referenceDataService)
+        {
+            _coursesApiClient = coursesApiClient;
+            _levyTransferMatchingService = levyTransferMatchingService;
+            _referenceDataService = referenceDataService;
+        }
+
+        public async Task<GetApplicationsForDownloadQueryResult> Handle(GetApplicationsForDownloadQuery request, CancellationToken cancellationToken)
+        {
+            var applications = await _levyTransferMatchingService.GetApplications(new GetApplicationsRequest
+            {
+                AccountId = request.AccountId,
+                PledgeId = request.PledgeId
+            });
+
+            if (!applications.Applications.Any())
+            {
+                return new GetApplicationsForDownloadQueryResult
+                {
+                    Applications = new List<ApplicationForDownloadModel>()
+                };
+            }
+
+            var standardTasks = RetrieveStandardTasks(applications);
```

Comment:
Standard info can be retrieved from the application directly now (from inner api)

Reply:
removed the file now